### ISSUE TITLE
Fine-tune the base model

### ISSUE THREAD

**Issue by shiliu0111** [prs-eth/Marigold#101]
Dear Authors,

Thank you for presenting this model. It is indeed a good model for downstream tasks. I have used your model for 3D reconstruction on my own data, and it performed well. As the reconstructed depth values will be used for a future regression task, I am seeking ways to improve accuracy. Could you provide any guidelines for fine-tuning?

Thanks in advance, and I wish you great success in the future!

**Comment by markkua** [comment]
Hi, if you have some training data from the interest domain with dense ground truth, you can fine-tune using the same training method. The most important details are described in the paper and supplementary sessions.
